fuzz/fuzzers/unbuffered.rs

```diff
@@ -0,0 +1,95 @@
+#![no_main]
+#[macro_use]
+extern crate libfuzzer_sys;
+extern crate rustls;
+
+use std::sync::Arc;
+
+use rustls::client::UnbufferedClientConnection;
+use rustls::server::{ResolvesServerCert, UnbufferedServerConnection};
+use rustls::unbuffered::{ConnectionState, UnbufferedStatus};
+use rustls::{ClientConfig, RootCertStore, ServerConfig, SideData};
+
+fuzz_target!(|data: &[u8]| {
+    let mut data = data.to_vec();
+    match data.split_first_mut() {
```

Comment:
similar question here RE: MSB vs first byte.

Reply:
This comment falls away with my understanding from [the other thread](https://github.com/rustls/rustls/pull/2249#discussion_r1869969109).